flink-tests/src/test/java/org/apache/flink/test/checkpointing/utils/SavepointMigrationTestBase.java

```diff
@@ -87,17 +89,24 @@ protected static String getResourceFilename(String filename) {
 		return resource.getFile();
 	}
 
-	@Before
-	public void setup() throws Exception {
+	private Configuration getConfigurationSafe() {
```

Comment:
I think we can get rid of this method by moving the `miniClusterResource` initialization to the constructor which says that it can throw an `Exception`.